test/TestExpectations.json

```
@@ -1583,72 +1583,228 @@
     "parameters": ["firefox", "webDriverBiDi"],
     "expectations": ["PASS"]
   },
+  {
+    "testIdPattern": "[cookies.spec] Cookie specs Page.cookies should get a cookie",
+    "platforms": ["darwin"],
```

Let's enable all test and mark the once that fail makes it much easier to update the expectations if we add a feature/bugfix.
```suggestion
    "testIdPattern": "[cookies.spec] Cookie specs Page.cookies *,
   "platforms": ["darwin", "linux", "win32"],
```

---

good idea